zerver/views/tutorial.py

```diff
@@ -4,15 +4,16 @@
 from django.http import HttpRequest, HttpResponse
```

Comment:
hm, I think we don't want to interact with the current tutorial, since this and a few other things (e.g. hotspots) will replace the tutorial once complete.